service/service.go

```diff
@@ -33,27 +33,19 @@ type Service interface {
 type service struct {
 	conf        *config.Config
 	enclaveInfo *sgx.EnclaveInfo
+	sgx         sgx.Sgx
 
 	oracleAccount *panacea.OracleAccount
 	oraclePrivKey *btcec.PrivateKey
 
 	queryClient panacea.QueryClient
-	grpcClient  *panacea.GRPCClient
+	grpcClient  panacea.GRPCClient
 	subscriber  *event.PanaceaSubscriber
 	txBuilder   *panacea.TxBuilder
-	ipfs        *ipfs.IPFS
+	ipfs        ipfs.IPFS
 }
 
-func New(conf *config.Config) (Service, error) {
-	queryClient, err := panacea.LoadVerifiedQueryClient(context.Background(), conf)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load query client: %w", err)
-	}
-
-	return NewWithQueryClient(conf, queryClient)
-}
-
-func NewWithQueryClient(conf *config.Config, queryClient panacea.QueryClient) (Service, error) {
+func New(conf *config.Config, sgx sgx.Sgx, queryClient panacea.QueryClient) (Service, error) {
```

Comment:
When creating a service, you need to create a queryClient depending on its purpose (New or Load).